IdeasTracker/IdeasTracker/IdeasTrackerApp.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  IdeasTrackerApp.swift
+//  IdeasTracker
+//
+//  Created by Wen Yu Ge on 2024-04-19.
+//
+
+import SwiftUI
+
+@main
+struct IdeasTrackerApp: App {
+
+    var body: some Scene {
+        WindowGroup {
+            SplashView()
+                .environmentObject(Router())
+                .environmentObject(LoginViewModel())
+                .environmentObject(IdeasViewModel())
```

Comment:
`IdeasViewModel` doesn't need to be an environment object; it's only use in the `IdeasView`